2016-06-03.md

```diff
@@ -0,0 +1,34 @@
+# Improving Rust, Letting Go of Python, Open Finances, and More
+
+## Multiple language support
+
+### Ramping up Rust
+We've long been intrigued by the Rust Language, and have lately been pushing forward for a more native Rust support experience, including easy deploy from the `t2` command line tool.
+
+You can see the state of the project at the [tessel-rust](https://github.com/tessel/tessel-rust) repo. If you're testing it out, please let us know what you think! Issues, PRs, and conversation in the #rust-lang channel of the [Tessel Slack](//tessel-slack.herokuapp.com) are all welcome.
+
+### Backing away from Python
+Ever since our announcement of Python on Tessel 2 last year, we've been waiting eagerly for a crowd of Pythonistas to charge in and excitedly exclaim, "I want that! I want to make that happen!"
+
+However, we haven't seen it. The basic pathway for Python on Tessel 2 has been proven out, but we haven't heard from anyone who's interested in taking on full support - or even much interest for using it. So we're letting it go. You'll see it disappear from the homepage soon. We can always pick up [from here](https://github.com/tessel/tessel-python) if the enthusiasm arrives.
+
+## Sponsor Tessel's development on Open Collective
+In pursuit of better financial transparency, we've just joined Open Collective: a way for us to raise funds for development and visibly disburse them to contributors. Check out the [Tessel Open Collective page](https://opencollective.com/tessel)!
+
+See more about the Tessel Project's finances [here](https://github.com/tessel/project/blob/master/FINANCES.md).
+
+## Development updates
+* On Tessel's command line, [the latest releases](https://github.com/tessel/t2-cli/releases) include some patching and corrections, improvements to the crash reporter tool which helps us identify and debug issues, and some background work for multiple language support.
+* In the firmware, [recent releases](https://github.com/tessel/t2-firmware/releases) have improved Windows USB driver support and added a few features to GPIO use of the module ports.
+
+## Want more updates?
+You can always find our weekly steering committee notes [here](https://github.com/tessel/project/tree/master/meetings) for the cutting edge of Tessel Project updates.
+
+Want to write this newsletter? It's an easy way to stay on top of Tessel development and be a participating member of the community. Reach out to @kelsey on [Tessel Slack](tessel-slack.herokuapp.com) if you're interested.
+
+## Giving a talk, or want to?
+Find opportunities and make announcements on the #events-speaking channel of the [Tessel Slack](tessel-slack.herokuapp.com). Join Tessel's community of speakers!
```

Comment:
There should definitely be a link to @johnnyman727's recent talk, as well as upcoming conferences/workshop Team Members will be attending.